nengo/utils/tests/test_stdlib.py

```diff
@@ -208,6 +211,44 @@ def test_weakkeydict_update():
         assert v is d.get(k)
 
 
+def test_weakkeyiddict_contains():
+    d = WeakKeyIDDictionary()
+    assert (None in d) is False
+
+
+def test_weakkeyiddict_iter_functions():
+    """Tests iterkeys and iteritems"""
+    d = WeakKeyIDDictionary()
+    in_d = {C(): 1, C(): 2, C(): 3}
+    d.update(in_d)
+    assert repr(d.iterkeys()).startswith(repr(d._keyrefs.values())[0:43])
+
+    results = list(d.iteritems())
+    start = "(<nengo.utils.tests.test_stdlib.C object "
+    end = [
+        "1)",
+        "2)",
+        "3)",
+    ]
+    end2 = [  # ending for travis-ci
```

Comment:
So the reason there's a difference in TravisCI is likely because in Python <= 3.5, dictionary items do not necessarily appear/iterate in the order that they were added. We should modify this test so that it's not dependent on dictionary order (e.g. just asserting that the expected items appear in the dictionary, not that they appear in a particular order).